src/ui/pages/NotificationDetails/components/ReceiveCredential.tsx

```diff
@@ -0,0 +1,169 @@
+import { IonCol, IonIcon } from "@ionic/react";
+import {
+  checkmark,
+  personCircleOutline,
+  swapHorizontalOutline,
+} from "ionicons/icons";
+import { useState } from "react";
+import "./ReceiveCredential.scss";
+import KeriLogo from "../../../assets/images/KeriGeneric.jpg";
+import { ResponsivePageLayout } from "../../../components/layout/ResponsivePageLayout";
+import { i18n } from "../../../../i18n";
+import { Alert as AlertDecline } from "../../../components/Alert";
+import { PageFooter } from "../../../components/PageFooter";
+import { useIonHardwareBackButton } from "../../../hooks";
+import { BackEventPriorityType } from "../../../globals/types";
+import { useAppDispatch, useAppSelector } from "../../../../store/hooks";
+import { PageHeader } from "../../../components/PageHeader";
+import { getConnectionsCache } from "../../../../store/reducers/connectionsCache";
+import { Agent } from "../../../../core/agent/agent";
+import { NotificationDetailsProps } from "../NotificationDetails.types";
+import {
+  getNotificationsCache,
+  setNotificationsCache,
+} from "../../../../store/reducers/notificationsCache";
+
+const ReceiveCredential = ({
+  pageId,
+  activeStatus,
+  notificationDetails,
+  handleBack,
+}: NotificationDetailsProps) => {
+  const dispatch = useAppDispatch();
+  const notificationsCache = useAppSelector(getNotificationsCache);
+  const [notifications, setNotifications] = useState(notificationsCache);
+  const connectionsCache = useAppSelector(getConnectionsCache);
+  const fallbackLogo = KeriLogo;
+  const [alertDeclineIsOpen, setAlertDeclineIsOpen] = useState(false);
+  const [initiateAnimation, setInitiateAnimation] = useState(false);
+  const ANIMATION_DELAY = 2000;
+  const connection = connectionsCache.filter(
+    (connection) => connection.id === notificationDetails.connectionId
+  )[0]?.label;
+
+  useIonHardwareBackButton(
```

Comment:
Why doesn't this apply to the multi-sig page?

Reply:
Sorry I missed that 👍